NativeScript/runtime/DataWrapper.mm

```diff
@@ -0,0 +1,81 @@
+#include "DataWrapper.h"
+
+#include "Caches.h"
+#include "FFICall.h"
+#include "ObjectManager.h"
+
+namespace tns {
+
+StructWrapper::~StructWrapper() {
+    parent_.Reset();
+    if (data_) {
+        // v8::Isolate::GetCurrent() may not necessarily be the isolate this object lives in.
```

Comment:
(but I sure hope it is.. I mean really, it should be.. if it's not, that seems like a v8-cppgc bug?)